woodwork/tests/accessor/test_column_accessor.py

```diff
@@ -761,7 +761,7 @@ def test_accessor_shallow_equality(sample_series):
     assert not metadata_col.ww.__eq__(diff_metadata_col.ww, deep=True)
 
     schema = metadata_col.ww.schema
-    diff_data_col = metadata_col.replace({"a": "1"})
+    diff_data_col = metadata_col.replace(to_replace="a", value="1")
```

Comment:
Probably don't need this since this was a change for pandas 1.4.0

Reply:
The latest dependency tests were failing, I think because of the 1.4.0 changes, so I was trying to just add the minimal changes to get the tests passing - https://github.com/alteryx/woodwork/runs/4985070473?check_suite_focus=true#step:10:254.

Happy to cherry-pick one of the commits that made these changes or restricted the versions, if you think that'd make more sense here